packages/curriculum-parser-markdown/plugins/insight/enki-links.js

```diff
@@ -26,24 +26,24 @@ module.exports = function enkiLink() {
     const indexOfEnki = node.url.indexOf(ENKI_ROOT);
     if (indexOfEnki > -1) {
       const toParse = `https://${node.url.substring(indexOfEnki)}`;
-      const { pathname, origin } = new UrlParse(toParse);
+      const { pathname, host } = new UrlParse(toParse);
       const { context, path } = getContextAndPath(pathname);
       if (CONTEXTS.includes(context)) {
         Object.assign(node, {
           isInternal: true,
           context,
           path,
-          origin,
+          host,
         });
       }
     }
   }
 
   function getContextAndPath(pathname) {
-    const firstSlashIndex = pathname.indexOf('/');
+    const [context, ...path] = pathname.split('/').filter(Boolean);
```

Comment:
`pathname` was parsed as `/glossary/general/oop`, such that `indexOf('/')` was always 0